pkgs/top-level/python-packages.nix

```diff
@@ -17041,6 +17041,29 @@ in modules // {
     };
   };
 
+  parsel = pkgs.python35Packages.buildPythonPackage rec {
+    name = "parsel-${version}";
+    version = "1.0.3";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/p/parsel/${name}.tar.gz";
+      sha256 = "9c12c370feda864c2f541cecce9bfb3a2a682c6c59c097a852e7b040dc6b8431";
+    };
+
+    buildInputs = with pkgs.python35Packages; [ pytest ];
+    propagatedBuildInputs = with pkgs.python35Packages; [ six w3lib lxml cssselect ];
+
```

Comment:
yeah, I forgot to make some changes... I tested this only with Python3.5, but ideally should work with all recent versions

I was hoping to catch failures with other Pythons with the CI